scalafmt-core/shared/src/main/scala/org/scalafmt/config/SpacesInInterpolatedStringCurlyBraces.scala

```diff
@@ -0,0 +1,39 @@
+package org.scalafmt.config
+
+import metaconfig._
+import org.scalafmt.config.SpacesInInterpolatedStringCurlyBraces._
+
+sealed abstract class SpacesInInterpolatedStringCurlyBraces
```

Comment:
i would suggest moving this class/object inside `object Spaces`, just like we did with `Newlines`.